perfkitbenchmarker/linux_packages/mongodb_server.py

```diff
@@ -18,6 +18,7 @@
 
 def YumInstall(vm):
   """Installs the mongodb package on the VM."""
+  vm.RemoteCommand('sudo setenforce 0')
```

Comment:
Thanks and happy to take the change.  Just want to make sure we are not
unduly exposing the user.  Then again we so require sudo everywhere...